docs/spec_draft.md

```diff
@@ -483,6 +484,61 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.cholesky
+
+### Semantics
+
+Computes the Cholesky decomposition of a batch of matrices.
+
+More formally, for all `i`, `result[i0, ..., iR-3, :, :]` is a Cholesky
+decomposition of `a[i0, ..., iR-3, :, :]`, in the form of either of a
+lower-triangular (if `lower` is `true`) or upper-triangular (if `lower` is
+`false`) matrix. The output values in the opposite triangle, i.e. the strict
+upper triangle or strict lower triangle correspondingly, are
+implementation-defined.
+
+If there exists `i` where the input matrix is not an Hermitian
+positive-definite matrix, then the result is implementation-defined.
+
+### Inputs
+
+| Name    | Type                                       |
+|---------|--------------------------------------------|
+| `a`     | tensor of floating-point or complex type   |
+| `lower` | 0-dimensional tensor constant of type `i1` |
```

Comment:
`lower` being a optional attribute, how about calling it in constants section  like
* (C3) `lower` is optional with default value `false`.